makefile.shade

```diff
@@ -0,0 +1,53 @@
+use import="Environment"
+
+var VERSION='0.1'
+var FULL_VERSION='0.1'
+var AUTHORS='Microsoft Open Technologies, Inc.'
+
+use-standard-lifecycle
+k-standard-goals
+
+var Configuration_Local = '${E("Configuration")}'
+default BASE_DIR_LOCAL='${Directory.GetCurrentDirectory()}'
+default TARGET_DIR_LOCAL='${Path.Combine(BASE_DIR_LOCAL, "artifacts")}'
+default BUILD_DIR_LOCAL='${Path.Combine(TARGET_DIR_LOCAL, "build")}'
+default SRC_PROJECT_GLOB_LOCAL="src/*/project.json"
+
+#build-compile target='compile' if='Directory.Exists("src")'
```

Comment:
In which case, can you simplify this? `AddAssemblyInfo` wouldn't apply so it could be reduced to `srcProject.ForEach(DotnetPack)`